app/routes/public/cfs/index.js

```diff
@@ -48,13 +48,26 @@ export default Route.extend({
       const userSession = await this.get('authManager.currentUser').query('sessions', {
         filter: [
           {
-            name : 'event',
-            op   : 'has',
-            val  : {
-              name : 'identifier',
-              op   : 'eq',
-              val  : eventDetails.id
-            }
+            and: [
+              {
+                name : 'event',
+                op   : 'has',
+                val  : {
+                  name : 'identifier',
+                  op   : 'eq',
+                  val  : eventDetails.id
+                }
+              },
+              {
+                name : 'speakers',
+                op   : 'any',
+                val  : {
+                  name : 'email',
+                  op   : 'eq',
+                  val  : this.get('authManager.currentUser').email
```

Comment:
```js
 this.get('authManager.currentUser')
```
has been used 3-4 times in the route, instead store it in a variable called `currentUser` instead of using `this.get` every time.